src/index.html

```diff
@@ -7,6 +7,38 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar calendar--length_29">
+      <div class="calendar__day calendar__day--start-day_sun"></div>
+      <div class="calendar__day"></div>
```

Comment:
`calendar__day` should be in `calendar` element

Reply:
calendar__day in calendar block as it shuold.